cli/internal/packager/deploy.go

```diff
@@ -82,6 +82,16 @@ func Deploy() {
 	}
 
 	if config.IsZarfInitConfig() {
+		// Create a gitea org and a read-only user
+		err := git.CreateZarfOrg()
```

Comment:
This org isn't being used currently. If it's dead code it should be removed.

I was originally going to push the repos to zarf org instead of pushing to `zarf-git-user` but that would need more Gitea api calls to create an empty repo before we push the first time (totally possible but wanted to try this out to validate first.)

@jeff-mccoy @RothAndrew any opinions?

Reply:
The org was only needed because I assumed we couldn't grant another user access to a private user repo, I saw let's KISS and just drop that requirement.